corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -203,6 +203,9 @@ hqDefine("app_manager/js/details/case_claim", function () {
         };
 
         subscribeToSave(self, searchConfigKeys, saveButton);
+        $(".case-search-multimedia-input button").on("click", function () {
```

Comment:
If someone is using a custom multimedia path, and they just change the path, will the save button get triggered? There might be a generic handler that listens for changes on all `<input>` elements, I can't remember.

Reply:
Good catch. Changing just the path does not enable save. I will take a look.

Reply:
resolved by https://github.com/dimagi/commcare-hq/pull/29765/commits/1541eb65b5f80484d7e6ffd9336ef72589465119